custom-configurations/src/main/java/com/sequenceiq/cloudbreak/domain/CustomConfigurationProperty.java

```diff
@@ -27,20 +33,22 @@
     private String name;
 
     @Column(nullable = false)
-    private String value;
+    @Convert(converter = SecretToString.class)
+    @SecretValue
+    private Secret value = Secret.EMPTY;
 
     @Column
     private String roleType;
 
     @Column(nullable = false)
     private String serviceType;
 
-    @ManyToOne
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
is the `CascadeType.ALL` intentional? will all functionality regarding custom config work in this case too?

Reply:
Yes, `CascadeType.ALL` is required to trigger the `saveAll` point for `SecretAspects` via `customConfigurationPropertyRepository` to persist/delete `CustomConfigurations` and `CustomConfigurationProperty`

Reply:
ok, thanks for the answer!